culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+This post proposes a system of **trust levels** for SourceCred the
+system, which describe how we intend to handle the very real questions
+of adversarial actors and gaming resistance over different time scales.
+
+The ideas proposed herein have been discussed informally within our
+community, but with somewhat confusingly varying semantics. It may help
+to have this shared reference to point to.
+
+**Note:** The term *trust levels* as used in this post is largely
+separate from [Discourse trust levels][discourse-trust], which are a
+construct of our forum software and not related to the SourceCred
+protocol in any way.
```

Comment:
Oh that's really interesting - I'm familiar with Discourse trust level and just assumed they were similar. Could this be confusing? Or would there be any case for overlap? (e.g., given that some plugin has trust levels, how to do those trust levels map to SourceCred? Or should they not?)